test/concrete/flow/FlowConstructionInitializeTest.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: CAL
+pragma solidity ^0.8.18;
+
+import {Vm} from "forge-std/Test.sol";
+
+import {InterpreterMockTest} from "test/abstract/InterpreterMockTest.sol";
+import {IFlowV5} from "src/interface/unstable/IFlowV5.sol";
+import {EvaluableConfigV3} from "rain.interpreter.interface/interface/IInterpreterCallerV2.sol";
+import {Flow} from "src/concrete/basic/Flow.sol";
+import {CloneFactory} from "rain.factory/src/concrete/CloneFactory.sol";
+import {FlowUtilsAbstractTest} from "test/abstract/FlowUtilsAbstractTest.sol";
+
+contract FlowConstructionInitializeTest is InterpreterMockTest, FlowUtilsAbstractTest {
+    CloneFactory internal immutable iCloneFactory;
+    IFlowV5 internal immutable iFlowImplementation;
+
+    constructor() {
```

Comment:
this dupes https://github.com/rainlanguage/rain.flow/pull/133/files abstract contract here, we can just inherit it